templates/telemetry_derived/experiments_v1/get_experiment_list.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+"""Get experiment list for a given date from the recipe server."""
+
+from argparse import ArgumentParser
+import json
+import sys
+import time
+
+import requests
+
+experiments_qualifying_actions = {"preference-experiment", "opt-out-study"}
+excluded_experiments = {
+    None,
+    "pref-flip-screenshots-release-1369150",
+    "pref-flip-search-composition-57-release-1413565",
+    "pref-hotfix-tls-13-avast-rollback",
+}
+recipes_url = "https://normandy.services.mozilla.com/api/v1/recipe/?format=json"
+timestamp_format = "%Y-%m-%dT%H:%M:%S.%f%z"
+
+parser = ArgumentParser(description=__doc__)
+parser.add_argument(
+    "date",
+    type=lambda value: time.strptime(value, "%Y-%m-%d"),
+    help="minimum last_updated date for disabled recipes",
+)
+
+
+def get_experiment_list(date):
+    for recipe in requests.get(recipes_url).json():
+        arguments = recipe.get("arguments") or {}
+        branches = arguments.get("branches")
+        name = arguments.get("slug", arguments.get("name"))
+        last_updated = time.strptime(recipe.get("last_updated"), timestamp_format)
+        if (
+            "branches" in arguments
```

Comment:
I'm confused at this clause (this will exclude old-style addon experiments, I'm fairly certain.) I can't seem to find a corresponding clause in TBV job either. Did this get added accidentally?

Reply:
in scala we decode this json into `case class NormandyRecipeArguments(branches: List[NormandyRecipeBranch], ...)`, which doesn't make `branches` optional, so `Try(json.extract[NormandyRecipe])` doesn't return a `Success`.

This is why we don't have `"Pioneer Enrollment"` in the current output.

If that behavior is a bug, i'm happy to remove this clause.